recipes/wg21-linear_algebra/all/conanfile.py

```diff
@@ -34,40 +34,45 @@ def _minimum_compilers_version(self):
             "clang": "8",
             "apple-clang": "11"
         }
+    
+    @property
+    def _full_compiler_version(self):
+        compiler = self.settings.compiler
+        if compiler == "msvc":
+            if compiler.update:
+                return int(f"{compiler.version}{compiler.update}")
+            else:
+                return int(f"{compiler.version}0")
+        else:
+            return compiler.version
 
-    def validate(self):
+    def configure(self):
```

Comment:
Seems like the change to ``configure`` is not needed here. This logic is better suited to the original ``validate`` (Unless I'm missing something!)